lib/ex_unit/lib/ex_unit/capture_io.ex

```diff
@@ -148,14 +148,19 @@ defmodule ExUnit.CaptureIO do
 
   See `capture_io/1` for more information.
   """
-  @spec capture_io(atom() | String.t() | keyword(), (-> any())) :: String.t()
-  def capture_io(device_input_or_options, fun)
+  @spec capture_io(atom() | pid() | String.t() | keyword(), (-> any())) :: String.t()
+  def capture_io(device_pid_input_or_options, fun)
```

Comment:
I was on the fence on whether to collapse these into fewer clauses. I can do that if you think it improves readability.